dashboard/src/components/Codeblock.tsx

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable solid/no-innerhtml */
+import { codeToHtml } from "shiki";
```

Comment:
```suggestion
import { codeToHtml } from "shiki";
```

This line specifically is causing the issue with shiki